### PR TITLE
Add support buton as icon on the top header

### PR DESCRIPTION
**Pull request by lukapr**
image

### CODE REVIEW

File: src/app/ui/common/support-modal/support-modal.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    (click)="modal.show()" title="Need support?">
+    <i class="fas fa-headset"></i>
+  </button>
+  <app-modal #modal>
+    <div class="app-modal-header">
+      Need support?
+    </div>
+    <div class="app-modal-body">
+      <div class="form-group">
+        <label for="exampleFormControlTextarea1">If you have any issues or need support please add the
```
Comment (ikruglik) [inline src/app/ui/common/support-modal/support-modal.component.html]: Do not see the mentioned id this label is for

File: src/app/ui/common/support-modal/support-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+})
+export class SupportModalComponent implements OnInit {
+
+  @ViewChild(ModalComponent) modal: ModalComponent;
+  
+  emailSupportText: string = "";
+
+  constructor(public supportService: SupportService) { }
+
+  ngOnInit() {
```
Comment (ikruglik) [inline src/app/ui/common/support-modal/support-modal.component.ts]: Empty block, can be removed

File: src/app/ui/common/support-modal/support-modal.component.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  
+  sendSupportEmail() {
+    this.supportService.sendSupportEmail(this.emailSupportText).subscribe(
+        (val) => {
+            console.log("POST call successful value returned in body",
+                val);
+        },
+        response => {
```
Comment (ikruglik) [inline src/app/ui/common/support-modal/support-modal.component.ts]: Please rename at least to errorResponse an use surrounding brackets

File: src/styles.css [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
 @import "~bootstrap/dist/css/bootstrap.css";
-@import "~font-awesome/css/font-awesome.css";
+@import "~@fortawesome/fontawesome-free/css/all.css";
```
Comment (ikruglik) [inline src/styles.css]: What is the difference between the libraries? Is it the proper way of importing it?

Comment (lukapr) [reply src/styles.css]: The new imported library is the v5 of fontawesome contains the new icons. Yeah, this is correct way of importing.

**Pull request merged**